content/en/docs/installation/armory-operator/op-quickstart.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Install Operator and Deploy Spinnaker Quickstart
+linkTitle: Quickstart
+weight: 1
+description: >
+  Install the Operator, create a Kubernetes manifest for Spinnaker, and then deploy using the Operator.
+---
+
+{{< include "armory-operator/os-operator-blurb.md" >}}
+
+## {{% heading "prereq" %}}
+
+Ensure you meet the following requirements:
+
+* You are familiar with [Kubernetes Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), which use custom resources to manage applications and their components.
+* You understand the concept of [managing Kubernetes resources using manifests](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/).
+* Your Kubernetes cluster runs version 1.13 or later. If you do not have a cluster already, consult guides for [Google](https://cloud.google.com/kubernetes-engine/docs/quickstart), [Amazon](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html), or [Microsoft](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal) clouds.
+* You have enabled admission controllers in Kubernetes (`-enable-admission-plugins`).
+* You have `ValidatingAdmissionWebhook` enabled in `kube-apiserver`. Alternatively, you can pass the `--disable-admission-controller` parameter to the to the `deployment.yaml` file that deploys the Operator.
```

Comment:
What if we added this to a kind of "Advanced" section? I doubt anyone using the managed offerings like EKS will know about these flags